R/00-setup_data.R

```diff
@@ -40,7 +42,7 @@ setup_data <- function(counts = NULL, pg_ids = NULL, pg_metadata = NULL, sample_
   # If they don't give sample metadata, then we will make up a row id
   if (is.null(sample_metadata)) sample_metadata <- data.frame(id = 1:nrow(counts))
   if (!is.data.frame(sample_metadata)) stop("metadata can only be in the form of a data.frame")
-  if (nrow(sample_metadata) != ncol(counts)) stop("the number of rows in the sample metadata is not equal to the number of columns in the counts" )
+  if (nrow(sample_metadata) != ncol(counts)) stop("the number of rows in the sample metadata is not equal to the number of columns in the counts")
```

Comment:
I also ran this through `styler` R package -- so you will see a lot of little things like spacing pop up -- not me going through manually to do this, but you can feel free to use styler on your own if you want (I am too lazy to style on my own 😄 )